client/human/HumanClientApp.cpp

```diff
@@ -200,6 +201,10 @@ void HumanClientApp::AddWindowSizeOptionsAfterMainStart(OptionsDB& db) {
     db.Add("video.windowed.top", UserStringNop("OPTIONS_DB_APP_TOP_WINDOWED"),    DEFAULT_TOP,         RangedValidator<int>(-max_height_plus_one, max_height_plus_one));
 }
 
+std::string HumanClientApp::EncodeServerAddressOption(const std::string& server) {
+    return "network.server.cookie._" + boost::replace_all_copy(server, ".", "_");
```

Comment:
are / will ipv6 addresses be supported? if so, also replacing `:` with `_` would be good.

Reply:
Looks like `:` works even without replacing. Still the only mention is:

> Avoid ":". Colons are reserved for namespaces (more later).

I've replaced it too.